conf/evolutions/default/3.sql

```diff
@@ -0,0 +1,16 @@
+# --- Indexing on queue for seach by queue feature
+# --- !Ups
+
+alter table yarn_app_result add column resource_used    BIGINT        UNSIGNED DEFAULT 0    COMMENT 'The resources used by the job in MB Seconds';
```

Comment:
I think the wording used is a bit confusing. Resources tend to be referred to as a generic thing when really we're talking about a specific type of resource (e.g. memory, CPU, storage, etc.) Maybe we should consider rewording stuff. For example, we might say "Total memory used over the job's duration (MB/s)", "Total memory wasted over the job's duration (MB/s)", etc. Let me know what you think.